clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -333,6 +379,17 @@ private EhcacheEntityResponse validate(ValidateCacheManager message) {
    * @return an {@code EhcacheEntityResponse} indicating the success or failure of the store creation operation
    */
   private EhcacheEntityResponse createServerStore(ClientDescriptor clientDescriptor, CreateServerStore createServerStore) {
+    ClientState clientState = this.clientStateMap.get(clientDescriptor);
+    if (clientState == null) {
+      return failure(new IllegalStateException("Client " + clientDescriptor + " is not connected to the Clustered Store Manager"));
+    }
+    if (!clientState.isAttached()) {
+      return failure(new IllegalStateException("Client " + clientDescriptor + " is not attached to the Clustered Store Manager"));
+    }
+    if (!isConfigured()) {
+      return failure(new IllegalStateException("Clustered Store Manager is not configured"));
```

Comment:
Could you include the clientDescriptor in this message like, "while attempting to create server-side storage for client {}"